tests/units/classes/asserters/phpResource.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace mageekguy\atoum\tests\units\asserters;
+
+use
+	mageekguy\atoum,
+	mageekguy\atoum\asserter,
+	mageekguy\atoum\tools\diffs,
+	mageekguy\atoum\tools\variable
+;
+
+require_once __DIR__ . '/../../runner.php';
+
+class phpResource extends atoum\test
+{
+	public function testClass()
+	{
+		$this->testedClass->extends('mageekguy\atoum\asserters\variable');
+	}
+
+	public function test__construct()
+	{
+		$this
+			->given($this->newTestedInstance)
+			->then
+				->object($this->testedInstance->getGenerator())->isEqualTo(new asserter\generator())
+				->object($this->testedInstance->getAnalyzer())->isEqualTo(new variable\analyzer())
+				->object($this->testedInstance->getLocale())->isEqualTo(new atoum\locale())
+				->variable($this->testedInstance->getValue())->isNull()
+				->boolean($this->testedInstance->wasSet())->isFalse()
+
+			->given($this->newTestedInstance($generator = new asserter\generator(), $analyzer = new variable\analyzer(), $locale = new atoum\locale()))
```

Comment:
it works here: https://github.com/atoum/atoum/pull/537/files#diff-f1d31228248836643d3c15038bcb5e73R76